assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -168,6 +168,8 @@ const baseActions = {
 			if ( webdatastream ) {
 				registry.dispatch( STORE_NAME ).setWebDataStreamID( webdatastream._id );
 				registry.dispatch( STORE_NAME ).setMeasurementID( webdatastream.measurementId ); // eslint-disable-line sitekit/acronym-case
+			} else {
+				registry.dispatch( STORE_NAME ).setWebDataStreamID( WEBDATASTREAM_CREATE );
```

Comment:
Should we maybe also do this above if `PROPERTY_CREATE === propertyID` as well? Currently it just returns early.

Reply:
Updated to set `webDataStreamID` setting to `WEBDATASTREAM_CREATE` by default. If we find a matching web data stream, we will update the `webDataStreamID` setting to the correct value.